examples/tutorials/advance_model_config/03_train_steps.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<p>Use batchsize of 64, as in all the other tutorials</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>